package.json

```diff
@@ -26,16 +36,6 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-react": "^7.29.3",
-    "eslint-plugin-react-hooks": "^4.3.0",
```

Comment:
A couple of development dependencies were being placed in the `development` key, so I moved to `devDependencies` and re-executed `yarn install` which updated `yarn.lock`